pages/dashboard/DashboardHandler.inc.php

```diff
@@ -119,10 +119,11 @@ function index($args, $request) {
 						'param' => 'assignedTo',
 						'value' => [],
 						'filterType' => 'pkp-filter-autosuggest',
+						'component' => 'field-select-users',
 						'autosuggestProps' => [
 								'allErrors' => (object) [],
 								'apiUrl' => $request->getDispatcher()->url($request, ROUTE_API, $context->getPath(), 'users', null, null, ['roleIds' => [ROLE_ID_MANAGER, ROLE_ID_SUB_EDITOR]]),
-								'component' => 'field-autosuggest',
+								'component' => 'field-select-users',
```

Comment:
I _think_ that this `component` is not necessary. The only one you need is the one at line 122 above.